assembly.c

```diff
@@ -149,12 +149,14 @@ void construct_START(CODE *code) {
 }
 
 /*
-mov rax, 1
-mov rbx, 0
-int 0x80
- */
+mov rax, 0x3C ; sys_exit
+xor rdi, rdi
```

Comment:
I'd prefer to keep this as an explicit 'exit with code 0' rather than a `xor` for readability and possibly future expandability